gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationTest.java

```diff
@@ -64,6 +64,7 @@
           put("fs.gs.application.name.suffix", "");
           put("fs.gs.authorization.handler.impl", null);
           put("fs.gs.authorization.handler.properties.", ImmutableMap.of());
+          put("fs.gs.refresh.access-token.per.request.enable", false);
```

Comment:
Sort alphabetically and rename this property to `fs.gs.refresh.access.token.per.request.enable` for consistency